cime_config/testdefs/testmods_dirs/clm/Fates/shell_commands

```diff
@@ -1,3 +1,2 @@
 ./xmlchange CLM_BLDNML_OPTS="-no-megan" --append
 ./xmlchange BFBFLAG="TRUE"
-./xmlchange CLM_FORCE_COLDSTART="on"
```

Comment:
Is this variable deprecated? Unnecessary?

Reply:
I had initially thought this was reverting a mistake that I made during the transition from `fates_main_api` to `master`, but it looks like it was introduced a while back in https://github.com/ESCOMP/ctsm/commit/34d8a33b80528b49cd94cd76645eefac26acd11b, so I guess I was wrong.

That said, I think the idea was that since we have a cold start testmod we should remove the coldstart from the generic `Fates` testmod and just make everything use `FatesColdDef` (with a possible look towards the future where we use `Fates` testmod with initialization files I think).  Spurred by your question I went to double check and it looks like I overlooked `FatesPRT2` testmod.  I'll fix it to use `FatesColdDef`.

- [ ] Update `FatesPRT2` testmod to use `FatesColdDef`

Reply:
Yes, I suggested to @glemieux that the base Fates test-mod NOT have the cold-start embedded in it, so that tests that do a cold start could inherit the cold-start as a separate test mod to include (which allows the way the cold start is done to change). This also allows some tests to inherit from a test mod that is NOT a cold-start. If you embed the FATES base test-mod with a cold-start you only allow FATES tests to be cold-starts.

Reply:
@ekluzek, if there is no restart, or finitdat file involved, what happens differently when CLM_FORCE_COLDSTART="on" ?   Does the force coldstart setting tell the scripting to ignore the file, or to use a subset of its information or something?

Reply:
@rgknox CLM_FORCE_COLDSTART tells build-namelist to explicitly set finidat to blank so that no restart file will be used. And it won't use any of the default finidat files that are in namelist_defaults even if they match.  It will abort with a warning if you do try to set an finidat file and have CLM_FORCE_COLDSTART=="on". This only applies to a "startup" case, for a hybrid or branch case it will use the finidat or nrevsn file as provided there.

For FATES we don't have finidat files that can be used, so it doesn't really apply much. We don't have a mechanism to ignore part of the finidat file. Although there are specific variables that are allowed to be missing for a startup case, whereas a restart or branch requires all restart variables to be provided. But, that's a variable by variable choice and there aren't that many that are allowed. Does that help?

Reply:
yes, thanks!